backend/amd64/proc.ml

```diff
@@ -283,10 +323,28 @@ let max_arguments_for_tailcalls = 10 (* in regs *) + 64 (* in domain state *)
      Return value in rax or xmm0. *)
 
 let loc_external_results res =
-  let (loc, _ofs) = calling_conventions 0 0 100 100 not_supported 0 res in loc
+  let (loc, _ofs) =
+    calling_conventions
+      ~first_int:0
+      ~last_int:4
+      ~step_int:4
```

Comment:
it would be nice to have an explanation of why the step is `4` here...

Reply:
That is indeed a hack to get `rdx` as the second
register. Hopefully the comments will clarify it,
but we are also planning a refactoring for `Reg.t`
so that should not stay as ugly for too long.